httpserver/service_node.cpp

```diff
@@ -344,6 +306,22 @@ void ServiceNode::bootstrap_peers(const std::vector<sn_record_t>& peers) const {
     std::vector<std::shared_ptr<request_t>> batches =
         to_requests(std::move(data));
 
+    /// Attach signature and pubkey
```

Comment:
Maybe make this a function since you do the same in `bootstrap_swarms`?

Reply:
Done.